mappings/net/minecraft/entity/passive/BeeEntity.mapping

```diff
@@ -133,10 +150,19 @@ CLASS net/minecraft/unmapped/C_lqpxgfgh net/minecraft/entity/passive/BeeEntity
 			ARG 5 pauseWhenMobIdle
 	CLASS C_splyjjod PollinateGoal
 		FIELD f_csokawwo flowerPredicate Ljava/util/function/Predicate;
+		FIELD f_dozwtbsh HOVER_POS_OFFSET F
+		FIELD f_gjvqbzgz MAX_POLLINATION_TICKS I
+		FIELD f_iqpjgscb MIN_FIND_FLOWER_RETRY_COOLDOWN I
+		FIELD f_iszvhyho SPEED_MODIFIER F
 		FIELD f_jtdhaaxa nextTarget Lnet/minecraft/unmapped/C_xxfpupgv;
+		FIELD f_jvckrwcl MIN_POLLINATION_TICKS I
 		FIELD f_nbbwihde pollinationTicks I
 		FIELD f_oqfhzoyr lastPollinationTick I
 		FIELD f_qzugdyap ticks I
+		FIELD f_tewzgskl ARRIVAL_THRESHOLD D
+		FIELD f_tzoeesbk POSITION_CHANGE_PROBABILITY I
+		FIELD f_webowsfk HOVER_HEIGHT_WITHIN_FLOWER F
+		FIELD f_yukhsgic MAX_FIND_FLOWER_RETRY_COOLDOWN I
 		FIELD f_zbijhfic running Z
 		METHOD m_honokgup moveToNextTarget ()V
 		METHOD m_hxxiczej completedPollination ()Z
```

Comment:
If this method is checking for whether the pollination is complete it should be pollinationCompleted or isPollinationCompleted but this isn't a big deal if you choose to keep this lol